README.md

```diff
@@ -13,12 +13,12 @@ The command line (cli, shell) is the interface between you (the user) and the op
 So since this is your first module, what can you expect the next three weeks?
 
 ## Planning for Current Class
-| Week | Topic | Read | Homework | Review |
-| ---- | ----- | ---- |----------|--------|
-|0.|Prepare for first session|[Pre-reading](https://github.com/HackYourFuture/HTML-CSS/tree/master/Week0)|-|-|
-| 1. | DOM, HTML/CSS syntax | [Week 1 Reading](https://github.com/HackYourFuture/HTML-CSS/tree/master/Week1/README.md) |  [Homework week 1](https://github.com/HackYourFuture/HTML-CSS/tree/master/Week1/MAKEME.md) |[Review](https://github.com/HackYourFuture/HTML-CSS/tree/master/Week1/REVIEW.md)|
-| 2. | [CLI](https://github.com/HackYourFuture/CommandLine/blob/master/Lecture-1.md), Responsive design, media queries, developer tools | [Week 2 Reading](https://github.com/HackYourFuture/HTML-CSS/tree/master/Week2/README.md) | [Homework week 2](https://github.com/HackYourFuture/HTML-CSS/tree/master/Week2/MAKEME.md) ||[Review](https://github.com/HackYourFuture/HTML-CSS/tree/master/Week2/REVIEW.md)|
-| 3. | [CLI](https://github.com/HackYourFuture/CommandLine/blob/master/Lecture-2.md), Recap, useful resources on the web| [JavaScript Intro](https://github.com/HackYourFuture/JavaScript/tree/laurens_thomas/Week0) | [Homework week 3](https://github.com/HackYourFuture/HTML-CSS/tree/master/Week3/MAKEME.md) ||[Review](https://github.com/HackYourFuture/HTML-CSS/tree/master/Week3/REVIEW.md)|
+| Week | Topic                                    | Read                                     | Homework                                 | Review                                   |
+| ---- | ---------------------------------------- | ---------------------------------------- | ---------------------------------------- | ---------------------------------------- |
```

Comment:
can you fix the markup? This looks really messy. Like this:

| Week | Topic | Read | Homework | Review |
| ----- | ------ | ----- |------------|--------|

Reply:
ah I mean: | ---- | ----- | ---- |----------|------| (get out all the spaces you have now)

Reply:
I think this my editor that did this automatically, very annoying

Reply:
hm ok I will fix it myself :)